.github/workflows/rust_ci.yml

```diff
@@ -137,7 +136,6 @@ jobs:
       - name: Upload to codecov.io
         uses: codecov/codecov-action@v2.1.0
         with:
-          token: ${{secrets.CODECOV_TOKEN}}
           flags: ironfish-rust-wasm
 
       # fmt
```

Comment:
Will other people be able to upload to our code cov?

Reply:
It's not anything we configure in our codecov settings, looks like they use the GitHub Actions environment variables to tag the build appropriately:

https://github.com/codecov/codecov-action/issues/29

https://github.com/codecov/codecov-action/releases/tag/v1.0.6